### DISCUSSION TITLE
No active connection found

### DISCUSSION THREAD

**Discussion by towilliamstwilio** [mtxr/vscode-sqltools#1184] [General]
Hi - I'm using the Redshift driver. I can connect and see my database and all tables, but when I try to run a query, I constantly get the error 'no active connection found' even though I'm running the query from the active connection. It even gives me the option to disconnect from the active connection.

**Comment by gjsjohnmurray** [comment]
Assuming you are using the dedicated Redshift driver I suggest you raise your issue with its author.
